client/src/components/reportPage.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ReportPage = ({ handleClose, show }) => {
+  const showHideClassName = show ? '' : 'display-none';
+  return (
+    <div className={`up-modalWrapper up-${showHideClassName}`}>
+      <div className="up-modal">
+        <section className="up-modal-report">
+          <h1> Report Track For </h1>
+          <br />
+          <ul>
```

Comment:
I wonder if you could make a react component for this, or use an array, e.g.
```
  const violations = ['Copyright infringement', 'Privacy Violation', ... ];
  const violationsJSX = violations.map(violation => (<a href="#">{violation}</a>));
```
Then you can just render it all together as:
```
<ul>
  {violationsJSX}
</ul>
```

I would also probably remove the br, and use styling to add in space around my `li`s